src/main/java/com/nike/cerberus/command/rds/RestoreRdsSnapshotCommand.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2019 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.command.rds;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.nike.cerberus.command.Command;
+import com.nike.cerberus.operation.Operation;
+import com.nike.cerberus.operation.rds.RestoreRdsSnapshotOperation;
+import org.apache.commons.lang3.StringUtils;
+
+import static com.nike.cerberus.command.rds.CopyRdsSnapshotsCommand.COMMAND_NAME;
+import static com.nike.cerberus.domain.cloudformation.CloudFormationParametersDelegate.STACK_REGION;
+
+@Parameters(
+        commandNames = COMMAND_NAME,
+        commandDescription = "Restore RDS cluster snapshot in a specified region by updating cloudformation stack"
+)
+public class RestoreRdsSnapshotCommand implements Command {
+
+    public static final String COMMAND_NAME = "restore-rds-snapshot";
+
+    public static final String SNAPSHOT_IDENTIFIER = "--snapshot-identifier";
+
+    public static final String REGION_ENV_KEY = "RESTORE_RDS_SNAPSHOT_REGION";
```

Comment:
This is confusing, can we only allow this to be passed in via command args and remove the logic for overloading it with env vars?

Reply:
I don't think any other command follows this pattern

Reply:
It's the hacky way of potentially taking arguments from copy-snapshot command. Maybe there're better ways to do it or we can make the decision later.

Reply:
OK, I remember talking about this with you. Maybe we should briefly look into extending the Composite operation to pass along state between sub commands.

Reply:
That's an option. We can make a decision when we chain them together.

Reply:
They're removed for now.